MyCryptoHQ/MyCrypto
# Network fixes for deterministic wallets UI

🎉 🎉 🎉

## Description

Fixes issues with not using the previously selected network for deterministic wallets UI when adding an existing account. 

## Changes

Changed all components using `DeterministicWallets` in the `AddAccountFlow` to get and use network details from `NetworkSelectPanel` (previous step in flow).

### Reusable Code/Components


## Steps to Test

1. Add a new account 
2. Select a network (fx. RSK)
3. Select mnemonic phrase and login (or any other type that uses `DeterministicWallets`)
4. The wallet select UI should show correct DPath and network symbol



## Quality Assurance
